pandas/core/arrays/categorical.py

```diff
@@ -1255,14 +1255,19 @@ def shape(self):
 
         return tuple([len(self._codes)])
 
-    def shift(self, periods):
+    def shift(self, periods, fill_value=None):
         """
         Shift Categorical by desired number of periods.
 
         Parameters
         ----------
         periods : int
             Number of periods to move, can be positive or negative
+        fill_value : object, optional
+            The scalar value to use for newly introduced missing values.
```

Comment:
see my comment below, -1 is an implementation detail and is not need to be mentioned here

Reply:
removed